package.json

```diff
@@ -286,69 +297,58 @@
         "when": "editorTextFocus && vim.active && vim.use<C-pageup> && !inDebugRepl"
       },
       {
-        "key": "left",
-        "command": "extension.vim_left",
-        "when": "editorTextFocus && vim.active && !inDebugRepl"
-      },
-      {
-        "key": "right",
-        "command": "extension.vim_right",
-        "when": "editorTextFocus && vim.active && !inDebugRepl"
-      },
-      {
-        "key": "up",
-        "command": "extension.vim_up",
-        "when": "editorTextFocus && vim.active && !inDebugRepl && !suggestWidgetVisible && !suggestWidgetMultipleSuggestions"
-      },
-      {
-        "key": "down",
-        "command": "extension.vim_down",
-        "when": "editorTextFocus && vim.active && !inDebugRepl && !suggestWidgetVisible && !suggestWidgetMultipleSuggestions"
+        "key": "shift+G",
+        "command": "list.focusLast",
+        "when": "listFocus && !inputFocus"
       },
       {
-        "key": "j",
-        "command": "list.focusDown",
-        "when": "listFocus && !inputFocus"
+        "key": "shift+backspace",
+        "command": "extension.vim_shift+backspace",
+        "when": "editorTextFocus && vim.active && vim.use<shift+BS> && vim.mode == 'SearchInProgressMode' && !inDebugRepl"
       },
       {
-        "key": "k",
-        "command": "list.focusUp",
-        "when": "listFocus && !inputFocus"
+        "key": "cmd+left",
+        "command": "extension.vim_cmd+left",
+        "when": "editorTextFocus && vim.active && vim.use<D-left> && !inDebugRepl && vim.mode != 'Insert'"
       },
       {
-        "key": "o",
-        "command": "list.toggleExpand",
-        "when": "listFocus && !inputFocus"
+        "key": "cmd+right",
+        "command": "extension.vim_cmd+right",
+        "when": "editorTextFocus && vim.active && vim.use<D-right> &&  !inDebugRepl && vim.mode != 'Insert'"
       },
       {
-        "key": "l",
-        "command": "list.select",
-        "when": "listFocus && !inputFocus"
+        "key": "cmd+a",
+        "command": "extension.vim_cmd+a",
+        "when": "editorTextFocus && vim.active && vim.use<D-a> && !inDebugRepl && vim.mode != 'Insert'"
       },
       {
-        "key": "h",
-        "command": "list.collapse",
-        "when": "listFocus && !inputFocus"
+        "key": "cmd+c",
+        "command": "extension.vim_cmd+c",
+        "when": "editorTextFocus && vim.active && vim.use<D-c> && vim.overrideCopy && !inDebugRepl"
       },
       {
-        "key": "g g",
```

Comment:
I'm fairly certain we need this. When the explorer window is selected, pressing `gg` will take you to the first item in the list.

Reply:
Oooops I'm sorry... entirely I didn't care the explorer window and the `when` condition`.
Will restore it later.